### ISSUE TITLE
Skall permission skickas med i User?

### ISSUE THREAD

**Issue by miguelogren** [sharpness-se/signup6-fe#19]

**Comment by jgrape** [comment]
Jo, det skall den. Så att man kan anpassa användargränssnittet utseende efter om man är Admin eller inte. Vissa knapppar/actions skall bara synas för de som är Admin.

Inte viktigt just nu i skrivande stund då frontend bara är "read only" (inga existerade andminfunktioner).
